tasks/InsightAppSec/helpers/insightAppSecApi.ts

```diff
@@ -112,6 +112,71 @@ export default class InsightAppSecApi
         }.bind(this));
     }
 
+    public async getAppName(appID)
+    {
+        return new Promise(async function (resolve, reject)
+        {
+            let response;
+            let payload = {type: "APP", query: "app.id='" + appID + "'"};
+            let app;
+
+            try
+            {
+                response = await this.makeApiRequest(this.endpoint + "/search", "POST", payload);
```

Comment:
We can have multiple apps (and other resource) with the same name, so it might be best in the future to use Get by ID when we have an ID. Might be a good task for Sophie to make the updates.

Reply:
The standard workflow is to get by ID whenever possible, get by name is only still there to support existing pipelines where the App and scan config have only the name entered and not the ID like the newer pipelines will

Reply:
We can get by ID here though, right?